x/interchainquery/keeper/new_query_test.go

```diff
@@ -0,0 +1,109 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	_ "github.com/stretchr/testify/suite"
+
+	icqtypes "github.com/Stride-Labs/stride/x/interchainquery/types"
+)
+
+type NewQueryTestCase struct {
+	ctx          sdk.Context
+	module       string
+	connectionId string
+	chainId      string
+	queryType    string
+	request      []byte
+	period       sdk.Int
+	callbackId   string
+	ttl          uint64
+	height       int64
+}
+
+func (suite *KeeperTestSuite) SetupNewQuery(
+	ctx sdk.Context,
+	module string,
+	connectionId string,
+	chainId string,
+	queryType string,
+	request []byte,
+	period sdk.Int,
+	callbackId string,
+	ttl uint64,
+	height int64,
+) NewQueryTestCase {
+
+	return NewQueryTestCase{
+		ctx:          ctx,
+		module:       module,
+		connectionId: connectionId,
+		chainId:      chainId,
+		queryType:    queryType,
+		request:      request,
+		period:       period,
+		callbackId:   callbackId,
+		ttl:          ttl,
+		height:       height,
+	}
+}
+
+func (s *KeeperTestSuite) TestNewQuerySuccessful() {
+	ctx := s.Ctx
```

Comment:
Can you move this variable initialization into `SetupNewQuery`?

Reply:
yep good idea

Reply:
done